### DISCUSSION TITLE
HMA! Pro VPN Installation Failed

### DISCUSSION THREAD

**Discussion by didepanlayar** [Homebrew/discussions#5177] [Casks] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.2.11
ORIGIN: https://github.com/Homebrew/brew
HEAD: d7d4c8266210c024b93a450a7d357cec0b46a1bb
Last commit: 22 hours ago
Core tap JSON: 05 Mar 05:30 UTC
Core cask tap JSON: 05 Mar 05:30 UTC
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 4
HOMEBREW_NO_AUTO_UPDATE: set
Homebrew Ruby: 3.1.4 => /usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/3.1.4/bin/ruby
CPU: quad-core 64-bit broadwell
Clang: 14.0.0 build 1400
Git: 2.37.1 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 7.87.0 => /usr/bin/curl
macOS: 12.6.7-x86_64
CLT: 14.2.0.0.1.1668646533
Xcode: N/A
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

An error occurred at the final stage of installation, here are my steps during installation:

Step 1: `brew install --cask hma-pro-vpn`

Output:
```
==> Downloading https://s-mac-sl.avcdn.net/macosx/privax/HMA-Pro-VPN.dmg
######################################################################################### 100.0%
Warning: No checksum defined for cask 'hma-pro-vpn', skipping verification.
==> Installing Cask hma-pro-vpn
Cask hma-pro-vpn only provides a manual installer. To run it and complete the installation:
  open /usr/local/Caskroom/hma-pro-vpn/latest/Install\ HMA\!\ Pro\ VPN.app
🍺  hma-pro-vpn was successfully installed!
```

Step 2: `open /usr/local/Caskroom/hma-pro-vpn/latest/Install\ HMA\!\ Pro\ VPN.app`

Output:
```
The file /usr/local/Caskroom/hma-pro-vpn/latest/Install HMA! Pro VPN.app does not exist.
```

After I checked the file name was `Install HMA! Pro VPN.pkg` not `Install HMA! Pro VPN.pkg` and I tried again `open /usr/local/Caskroom/hma-pro-vpn/latest/Install\ HMA\!\ Pro\ VPN.pkg` and the problem occurred as in the picture below.

<div align="center">
    <img src="https://github.com/Homebrew/discussions/assets/64562038/5cc3ccbd-708a-496e-95e0-fe889eea93a6">
</div>

### Relevant casks

hma-pro-vpn

**Comment by SMillerDev** [comment]
Unfortunately Homebrew is not the manufacturer

**Reply by didepanlayar** [reply]
Of course I understand that, but when I downloaded it via the manufacturer with the following URL `https://s-mac-sl.avcdn.net/macosx/privax/HMA-VPN.dmg` the application was successfully installed.

However in the Homebrew repository it is downloaded via the url `https://s-mac-sl.avcdn.net/macosx/privax/HMA-Pro-VPN.dmg` so maybe this is what makes the difference.

So, I just want all the applications that I install to be able to use Homebrew, unless the application is not in Homebrew.

**Reply by daeho-ro** [reply]
Maybe no more pro version?

**Comment by daeho-ro** [comment]
I cannot find much details about pro plan, and so if it is changed to use hma vpn only, then the cask should be registered without pro tag.

**Reply by daeho-ro** [reply, marked as answer]
I create a PR and this may help you to install hma-vpn later.
https://github.com/Homebrew/homebrew-cask/pull/169298

**Reply by daeho-ro** [reply]
Now, you can install `hma-vpn`.

**Reply by didepanlayar** [reply]
Hi @daeho-ro , thanks for the update, now I have successfully installed `hma-vpn` with Homebrew.
